libioc/Config/Jail/Globals.py

```diff
@@ -61,16 +64,28 @@
     "allow_sysvipc": 0,
     "allow_raw_sockets": 0,
     "allow_chflags": 0,
+    "allow_mlock": None,
     "allow_mount": 0,
     "allow_mount_devfs": 0,
+    "allow_mount_fusefs": False,
     "allow_mount_nullfs": 0,
     "allow_mount_procfs": 0,
     "allow_mount_fdescfs": 0,
     "allow_mount_zfs": 0,
     "allow_mount_tmpfs": 0,
     "allow_quotas": 0,
     "allow_socket_af": 0,
+    "allow_tun": 0,
     "allow_vmm": False,
+    "available": 0,
+    "bpf": None,
+    "comment": None,
+    "compression": None,
+    "compressratio": None,
```

Comment:
This is not a property that can be set. It should not be in the jail configuration at all. Is that the case in python-iocage?

Reply:
Yes,  such parameters using with python-iocage.
https://www.freebsd.org/cgi/man.cgi?query=iocage&sektion=8

* available
https://www.freebsd.org/cgi/man.cgi?query=iocage&sektion=8

```
     available	   Available space in the jail's dataset.  The amount of space
		   available to	the dataset and	all its	children, assuming
		   that	there is no other activity in the pool.	 Because space
		   is shared within a pool, availability can be	limited	by any
		   number of factors, including	physical pool size, quotas,
		   reservations, or other datasets within the pool.

		   Source: zfs(8)
```

* bpf

https://github.com/iocage/iocage/blob/528f007e92f6e081614d8a09186c4831fc9f3e32/iocage.8#L1019

* compression

https://github.com/iocage/iocage/blob/528f007e92f6e081614d8a09186c4831fc9f3e32/iocage.8#L1724

```
Controls the compression algorithm used for this dataset.
The lzjb compression algorithm is optimized for performance while
providing decent data compression.
Setting compression to on uses the lzjb compression algorithm.
The gzip algorithm uses the same compression as the
```

* compressratio

https://github.com/iocage/iocage/blob/528f007e92f6e081614d8a09186c4831fc9f3e32/iocage.8#L1789

```
Compression ratio.
Read-only.
For non-snapshots, the compression ratio achieved for the used space
of this dataset, expressed as a multiplier.
The used property includes descendant datasets, and, for clones, does
not include the space shared with the origin snapshot.
```

Also, Also, `comment` property setting a lower layer for `user` property.

ref https://github.com/bsdci/libioc/pull/630#discussion_r253304269

For example
```
user {
  comment: None,
  notes: None
}